euphony/src/main/cpp/core/source/UTF8Charset.cpp

```diff
@@ -0,0 +1,26 @@
+#include "UTF8Charset.h"
```

Comment:
this include statement is working, but what about writing the file actual path for consistency? 😮

Reply:
Oh I see, Thanks for your feedback 😄